tests/dags/subdir2/test_dont_ignore_this.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from datetime import datetime
+
+from airflow.models import DAG
+from airflow.operators.bash_operator import BashOperator
+
+DEFAULT_DATE = datetime(2019, 12, 1)
+
+args = {
+    'owner': 'airflow',
+    'start_date': DEFAULT_DATE,
+}
+
+dag = DAG(dag_id='test_dag_under_subdir2', default_args=args)
```

Comment:
This doesn't actually need a dag for the test to work, so this file can be mostly empty.

Reply:
It can be almost empty but this is under the dags folder and we're making this file look like a dag to Airflow so it seems to make sense to place an actual test dag here.

Reply:
Yes and no - we're only testing the ignore behaviour, which we test "in isolation" outside of the rest of the scheduler. Anyway, let's put `schedule_interval=None` then please so that this doesn't actually run?